specutils/analysis/uncertainty.py

```diff
@@ -89,3 +89,92 @@ def _snr_single_region(spectrum, region=None):
     # the axis=-1 will enable this to run on single-dispersion, single-flux
     # and single-dispersion, multiple-flux
     return np.mean(flux / uncertainty, axis=-1)
+
+
+def snr_der(spectrum, region=None):
```

Comment:
I found the `der`pretty confusing here.  What about `snr_derived` (which is what the "der" seem to mean here) or `snr_nouncertainty`?  @nluetzge, do you have any opinion on that?

Reply:
I would stay away from `snr_nouncertainty` as there might be other algorithms that don't use the uncertainty. I like the  `snr_derived`, personally, as that is what it talked about in the PDF.

Reply:
How about  `snr_robust_estimate`?